### PR TITLE
quincy: doc/cephadm: add note about ceph-exporter (Quincy)

### PR DESCRIPTION
**Pull request by zdover23**
Explain how to deal with the error message "Non-zero exit code 22" when using cephadm to install a Ceph cluster.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/cephadm/install.rst [hunk 1]
```diff
@@ -148,6 +148,38 @@ bootstrap`` command, so you'll need to know the IP address of that host.
...
+
+   This command should result in a successfully-deployed cluster. Expand the
+   successfully deployed cluster by deploying the monitoring stack without
+   ``ceph-exporter`` by running the following commands:
+
+   .. prompt:: bash $
+
+      ceph orch apply prometheus
+      ceph orch apply grafana
+      ceph orch apply node-exporter
```
Comment (adk3798) [inline doc/cephadm/install.rst:175]: we need `ceph orch apply alertmanager` as well

**Review by adk3798 — APPROVED** [review]
LGTM, going to approve, but have one review comment that should be addressed.

File: doc/cephadm/install.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephadm/install.rst:175]: @adk3798 Roger that. It's done. Thank you for your quick attention.

File: doc/cephadm/install.rst [hunk 2]
```diff
@@ -148,6 +148,39 @@ bootstrap`` command, so you'll need to know the IP address of that host.
...
+
+   If you see this error message, add the ``--skip-monitoring-stack`` flag to
+   your ``cephadm bootstrap`` command. To do this, run a command of the
+   following form:
+
+   .. prompt:: bash $
+
+      cephadm bootstrap --mon-ip {monitor IP address} --skip-monitoring-stack
+
+   This command should result in a successfully-deployed cluster. Expand the
```
Comment (anthonyeleven) [inline doc/cephadm/install.rst:167]: s/Expand/Enhance/

**Review by anthonyeleven — APPROVED** [review]

File: doc/cephadm/install.rst [hunk 3]
```diff
@@ -148,6 +148,39 @@ bootstrap`` command, so you'll need to know the IP address of that host.
...
+
+   .. prompt:: bash $
+
+      ceph orch apply prometheus
+      ceph orch apply grafana
+      ceph orch apply node-exporter
+      ceph orch apply alertmanager
+
+   This error occurs because some Quincy releases contain a faulty
+   ``ceph-exporter``. After this fault was discovered, ``ceph-reporter`` was
```
Comment (nizamial09) [inline doc/cephadm/install.rst:179]:
```suggestion
   ``ceph-exporter``. After this fault was discovered, ``ceph-exporter`` was
```

Comment (zdover23) [reply doc/cephadm/install.rst:179]: Aw man. good catch, @nizamial09.

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**